src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -1790,8 +1790,6 @@ finish trigger, because the underlying member outputs must already be optional.
 Graph Branching
 ---------------
 
-.. versionadded:: 8.0.0
-
```

Comment:
Might be worth keeping this?

Reply:
But branching was still possible in Cylc 7? There is a note about the difference between 7 and 8 just below

Reply:
Fair enough.